### PR TITLE
Best practices Check

### PR DESCRIPTION
**Pull request by Alhajideen**
## To-do
Here I have added all the files from the to-do app for your review.
I await your feedback

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Alhajideen/webpack-boilerplate/issues"
+  },
+  "homepage": "https://Alhajideen.github.io/T0-Do-List",
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.2.0",
```
Comment (DebbyMiressa) [inline package.json:28]: This might be a bug! Try changing @fortawesome to @fontawesome

**Review by DebbyMiressa — CHANGES_REQUESTED** [review]
Hi @Alhajideen 👋 

Good job on your awesome project. But here is one thing you might want to improve to make your project flawless.

File: package.json (hunk 1 above)
Comment (Alhajideen) [reply package.json:28]: Hi @DebbyMiressa ,
I think it is not a bug.
That was added by default after I installed fontawesome on my project.

Thanks.

**Review by DebbyMiressa — APPROVED** [review]
Hi @Alhajideen 👋 

Your project is now complete! Well Done!!



Good Job👍 and cheers 🥂

**Pull request merged**